sdk/appconfiguration/Azure.ApplicationModel.Configuration/samples/Sample7_ConfiguringPipeline.cs

```diff
@@ -25,15 +25,12 @@ public void ConfiguringPipeline()
             options.Transport = new HttpClientTransport(s_client);
 
             // remove logging policy
-            options.LoggingPolicy = null;
+            options.Diagnostics.DisableLogging = true;
```

Comment:
This does not follow our boolean property guidelines. What about IsLoggingEnabled?